docs/_docs/04_reference/02-cli-options.md

```diff
@@ -35,67 +35,68 @@ This page documents the CLI commands and options available with Terragrunt:
   - [graph](#graph)
   - [stack](#stack)
 - [CLI options](#cli-options)
-  - [terragrunt-check](#terragrunt-check)
```

Comment:
Why was this updated?

Reply:
I forgot that I had done this. I tried to match the table of contents at the top with the order of stuff below. You can see that I've moved it rather than deleted it.